unity-client/Assets/UnityGLTF/Scripts/UniformMaps/SpecGloss2StandardMap.cs

```diff
@@ -45,6 +70,30 @@ public virtual int SpecularGlossinessTexCoord
             set { return; }
         }
 
+        private Vector2 specGlossOffset = Vector2.zero;
```

Comment:
All offsets that aren't directed to `_BaseMap` will not work because we aren't declaring the `_ST` uniforms for each one and implementing the corresponding offset UV transform. You can check that the only declared `_ST` is the `_BaseMap_ST` in `LitInput.hlsl`.

The UV transform for `_BaseMap` is computed here: https://github.com/decentraland/explorer/blob/e40d56c939db891b53342822a29da5adf7363d7d/unity-client/Assets/Rendering/Shaders/URP/LitForwardPass.hlsl#L95

I'm afraid that we might not have enough TEXCOORD semantics to support this extension entirely without removing lightmapping support and doing some gymnastics.

Reply:
Also, we should add at least a test with a GLTF cube/plane + offseted material to ensure this is working properly. We are supporting a new extension here, so it deserves the extra mile.

Reply:
Ah dammit... I mixed `_ST` with samplers and I thought it would work since said samplers are actually declared per texture.

Then I will remove the feature for anything that is not a basemap and talk with @nearnshaw so the documentation reflects it properly. I was also planning of doing a bunch of visual tests as well, I know they are disabled in the CI but they are more reliable to changes on the shaders.